### PR TITLE
Add send confirm existing account phone page for register journey

### PR DESCRIPTION
**Pull request by wilpow-github**
### Context

As an overseas qualified teacher
I want to sign in/register a Teaching services account
So that I can apply for QTS

### Changes proposed in this pull request

Add a new page at /sign-in/register/send-confirm-existing-account-phone

### Checklist

-   [x] Attach to Trello card
-   [x] Rebased master
-   [x] Cleaned commit history
-   [x] Tested by running locally
-   [x] Reminder created to manually clean any removed app settings post deployment

### CODE REVIEW

File: dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Redactor.cs [hunk 1]
```diff
@@ -55,4 +55,14 @@ public class Redactor
...
     }
+
+    public string? RedactMobileNumber(string mobileNumber)
+    {
+        if (string.IsNullOrEmpty(mobileNumber))
+        {
+            return string.Empty;
+        }
+
+        return new string('*', mobileNumber.Length - 4) + mobileNumber.Substring(mobileNumber.Length - 4);
```
Comment (gunndabad) [inline dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Redactor.cs:66]: Would be good to clone the implementation we're already using elsewhere: https://github.com/DFE-Digital/dqt-web-portal/blob/3811d7449b34f318d27f9e250f76a9181e890f1d/src/Dqt.Portals/Capgemini.DfE.Evolved.Model/RedactionHelper.cs#L71

Comment (wilpow-github) [reply dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Redactor.cs:66]: Would that work for international numbers? We're storing in the + format so the user will see e.g. +9************73. Could potentially cause confusion - especially if they entered as 0093...

Comment (gunndabad) [reply dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Redactor.cs:66]: Fair point. Let's keep it as you have it.

**Review by gunndabad — APPROVED** [review]

File: dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Redactor.cs (hunk 1 above)
Comment (wilpow-github) [reply dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Redactor.cs:71]: Added a case for when the mobile number is 4 or less

**Pull request merged**